Releasenotes.md

```diff
@@ -0,0 +1,77 @@
+  - PartnerhistorieHalCollectie__embedded --> PartnerhistorieHalCollectieEmbedded
+  - Partnerhistorie_links --> PartnerhistorieLinks
+  - VerblijfstitelhistorieHalCollectie__embedded --> VerblijfstitelhistorieHalCollectieEmbedded
+  - AangaanHuwelijkInOnderzoek --> AangaanHuwelijkPartnerschapInOnderzoek
+
+- Schema-component Burgerservicenummer is verwijderd
+
+- Enkele property-namen zijn gewijzigd
+  - nationaliteithistorie.datumEindeGeldigheid --> nationaliteithistorie.datumTot
+  - Verblijfplaatshistorie_links.nummeraanduiding --> VerblijfplaatshistorieLinks.adres
+  - Verblijfplaats is anders opgebouwd.
+    - Schema-component BinnenlandsAdres is verwijderd. Er wordt nu hergebruik gemaakt van het component Adres ipv het component BinnenlandsAdres
+      - BinnenlandsAdres.openbareRuimteNaam --> Verblijfplaats.straat _**(breaking)**_
+      - BinnenlandsAdres.functieAdres --> Verblijfplaats.functieAdres _**(breaking)**_
+      - BinnenlandsAdres.aanduidingBijHuisnummer --> Verblijfplaats.aanduidingBijHuisnummer _**(breaking)**_
+      - BinnenlandsAdres.identificatiecodeNummeraanduiding --> Verblijfplaats.nummeraanduidingIdentificatie _**(breaking)**_
+      - BinnenlandsAdres.woonplaatsnaam --> Verblijfplaats.woonplaats _**(breaking)**_
+      - Verblijfplaats.identificatiecodeAdresseerbaarObject --> Verblijfplaats.adresseerbaarObjectIdentificatie _**(breaking)**_
+      - Verblijfplaats.straatnaam --> Verblijfplaats.korteNaam _**(breaking)**_
+      - component verblijfBuitenland is komen te vervallen. _**(breaking)**_
+    - Schema-component Verblijfbuitenland is verwijderd. Properties zijn opgenomen in Verblijfplaats.
+      - VerblijfBuitenland.adresRegel1 --> Verblijfplaats.adresregel1 _**(breaking)**_
+      - VerblijfBuitenland.adresRegel2 --> Verblijfplaats.adresregel2 (br _**(breaking)**_
+      - VerblijfBuitenland.vertrokkenOnbekendWaarheen --> Verblijfplaats.vertrokkenOnbekendWaarheen _**(breaking)**_
+      - VerblijfBuitenland.land --> Verblijfplaats.land _**(breaking)**_
+    - VerblijfplaatsInOnderzoek.identificatiecodeNummeraanduiding --> VerblijfplaatsInOnderzoek.nummeraanduidingIdentificatie _**(breaking)**_
+    - VerblijfplaatsInOnderzoek.identificatiecodeAdresseerbaarObject --> VerblijfplaatsInOnderzoek.adresseerbarObjectIdentificatie _**(breaking)**_
+    - VerblijfplaatsInOnderzoek.straatnaam --> VerblijfplaatsInOnderzoek.korteNaam _**(breaking)**_
+    - VerblijfplaatsInOnderzoek.naamOpenbareRuimte --> VerblijfplaatsInOnderzoek.straat _**(breaking)**_
+    - VerblijfplaatsInOnderzoek.woonplaatsnaam --> VerblijfplaatsInOnderzoek.woonplaats _**(breaking)**_
+
+- Toegevoegde properties:
+  - OntbindingHuwelijkInOnderzoek.reden is toegevoegd
+
+- Er is in de toelichting van Hallinks opgenomen dat een Hallink ook een templated link kan zijn.
+
+### Features:
+
+In het algemeen is de testdata verwijderd uit de feature-files. De feature files hebben als doel om functioneel de werking te beschrijven.
```

Comment:
hoe bedoel je dit? Welke testdata is er verwijderd? De data staat beschreven onder Gegeven in de features.

Of verwijs je naar het testdata bestand dat in BRP (actueel) stond. Die heeft denk ik nooit betrekking gehad op de historie features. Of heb je echt iets gewijzigd in de features hiervoor?